content/php/concepts/string-functions/terms/htmlspecialchars/htmlspecialchars.md

```diff
@@ -0,0 +1,95 @@
+|`ENT_HTML5`     | Handles code as HTML 5                                                                                               |
+|`ENT_XML1`      | Handles code as XML 1                                                                                                |
+|`ENT_XHTML1`    | Handles code as XHTML                                                                                                |
+
+- `$encoding`: An optional string value that specifies the character-set to use when converting characters. 
+
+The supported character sets are listed in the table below:
+
+| Charset     | Description                                                                             |
+|-------------|-----------------------------------------------------------------------------------------|
+|`UTF-8`      | The default. ASCII compatible multi-byte 8-bit Unicode                                  |
+|`ISO-8859-1` | Western European                                                                        |
+|`ISO-8859-15`| Western European (adds the Euro sign + French and Finnish letters missing in ISO-8859-1)|
+|`cp866`      | DOS-specific Cyrillic charset                                                           |
+|`cp1251`     | Windows-specific Cyrillic charset                                                      |
+|`cp1252`     | Windows specific charset for Western European                                          |
+|`KOI8-R`     | Russian                                                                                |
+|`BIG5`       | Traditional Chinese, mainly used in Taiwan                                              |
+|`GB2312`     | Simplified Chinese, national standard character set                                     |
+|`BIG5-HKSCS` | Big5 with Hong Kong extensions                                                          |
+|`Shift_JIS`  | Japanese                                                                                |
+|`EUC-JP`     | Japanese                                                                                |
+|`MacRoman`   | Character-set that was used by Mac OS                                                   |
+
+- `$double_encode`:  An optional boolean value that specifies whether to encode existing html or not. 
+
+**Note**: If `$string` contains an invalid code, the function will return an empty string unless `$flag` is set to `ENT_IGNORE` or `ENT_SUBSTITUTE`.
+
+## Example
+The following example uses `htmlspecialchars()` to convert the special character `&` in a string  to an HTML entity `&amp;`.
+```php
+<?php
+    $statement ="Codecademy  offers courses in various programing languages like Java, PHP, C++ & many others.";
+    echo htmlspecialchars($statement, ENT_COMPAT);
+?>
+```
+This results in the following output:
+```shell
+Codecademy  offers courses in various programing languages like Java, PHP, C++ &amp; many others.
+```
+## Codebyte Example
+
+```codebyte/php
```

Comment:
Need an introductory sentence for the codebyte summarizing what it is doing. You also need to change it so it isn't a copy of the prior example.

Reply:
Hi @SSwiniarski,  I have included the code-byte title and reworked the example.